examples/graphs/node2vec_movielens.py

```diff
@@ -0,0 +1,584 @@
+"""
+Title: Graph representation learning with node2vec.
+Author: [Khalid Salama](https://www.linkedin.com/in/khalid-salama-24403144/)
+Date created: 2021/05/15
+Last modified: 2021/05/15
+Description: Implementing the node2vec to generate embeddings for movies.
```

Comment:
Suggestion:

- Add "model"
- Include the dataset name (Movielens) to make sure users don't confuse this "movies" example with a computer vision example
LMKWYT 👍 

```suggestion
Description: Implementing the node2vec model to generate embeddings for movies from the Movielens dataset.
```